tests/module/module2.dora

```diff
@@ -0,0 +1,11 @@
+//= cannon
```

Comment:
FYI: The line `//= cannon` isn't necessary anymore, all tests are now run with both compilers. No strong need to remove this line here (although you can if you want). I plan to remove those lines in all tests anyway.

Reply:
Ok, thanks!